impeller/renderer/backend/gles/pipeline_library_gles.cc

```diff
@@ -257,6 +257,19 @@ PipelineFuture<ComputePipelineDescriptor> PipelineLibraryGLES::GetPipeline(
   return future;
 }
 
+// |PipelineLibrary|
+void PipelineLibraryGLES::RemovePipelinesWithEntryPoint(
+    std::shared_ptr<const ShaderFunction> function) {
+  for (auto it = pipelines_.begin(); it != pipelines_.end();) {
+    if (it->first.GetEntrypointForStage(function->GetStage())
+            ->IsEqual(*function)) {
+      it = pipelines_.erase(it);
+      continue;
+    }
+    it++;
```

Comment:
Why here and not as the last clause of the `for`?

Reply:
The `continue` would make them not identical.